### ISSUE TITLE
URL in LaTeX output should be wrapped with `\nolinkurl{}`

### ISSUE THREAD

**Issue by xuhdev** [sphinx-doc/sphinx#2383]
Currently the URL in the LaTeX output is not wrapped with `\nolinkurl{}`. As a result, the line breaking is not optimized: they may be hyphenated, which is very confusing for URLs.

**Comment by jfbu** [comment]
Which kind of URLs ? recently there was commit 0211e19a, Fix #2337: Use `\url{URL}` macro instead of `\href{URL}{URL}` in LaTeX writer.

**Comment by xuhdev** [comment]
You can have URLs in footnotes, if `latex_show_urls = 'footnote'`. The URLs in footnotes are not supposed to be clickable.

**Comment by jfbu** [comment]
ok. I had not encountered the problem because the URLs in footnotes were not long enough for  hyphenation to occur frequently enough for me to realize it.

**Comment by tk0miya** [comment]
Note:

Input:

```
`test <http://www.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.com>`_
```

Output:
![2016-05-28 15 16 56](https://cloud.githubusercontent.com/assets/748828/15625637/60d70004-24e9-11e6-9003-421233c2d0d6.png)

**Comment by tk0miya** [comment]
Fixed at b74b9cb.
Thanks,
